server/tso/allocator_manager.go

```diff
@@ -535,18 +535,25 @@ func (am *AllocatorManager) campaignAllocatorLeader(
 func (am *AllocatorManager) AllocatorDaemon(serverCtx context.Context) {
 	tsTicker := time.NewTicker(am.updatePhysicalInterval)
 	defer tsTicker.Stop()
-	patrolTicker := time.NewTicker(patrolStep)
-	defer patrolTicker.Stop()
-	checkerTicker := time.NewTicker(PriorityCheck)
-	defer checkerTicker.Stop()
+	var patrolTickerC, checkerTickerC <-chan time.Time
+	// Local TSO related daemon goroutines only work when enableLocalTSO is true.
+	if am.enableLocalTSO {
+		patrolTicker := time.NewTicker(patrolStep)
```

Comment:
`patrolTickerC = time.NewTicker(patrolStep)`?

Reply:
It's `patrolTickerC = patrolTicker.C` in line #542.